_build/templates/default/sass/index.scss

```diff
@@ -22,6 +22,7 @@ $fa-css-prefix: icon;
 
 $fa-css-prefix: fa;
 
+@import "normalize/_normalize";
```

Comment:
You have added the normalize-scss package to package.json but you have included the files of that package in this commit too (and this twice: i.e. normalize/normalize/_variables.scss and normalize/_variables.scss!). Adding the files is superfluous.

Please refer here to the files in this line by descending/ascending the tree with multiple `../` until you get to `node_modules/normalize-scss/sass/_normalize.scss` (please check the path).

Reply:
In the gruntfile.js I made similar to Font-awesome copying files from node_modules folder. Next is the connection of one normalize file.scss . The way I fixed it.

Reply:
But copying is not needed, since there are no files referenced by url like font files for the browser.

Reply:
Copying the scss files of fontawesome is too much too. The location of the font files could be set with a sass variable (if thats not done already) and the files could be included directly from the node_modules folder.

Reply:
Fixed

Reply:
Thanks!